README.md

```diff
@@ -72,14 +76,10 @@ inside your session. After this, if you want quit the Postgres client, simply ty
 
 --- 
 
-With the development database created and python requirements installed we can migrate the tables. We will use the `orator_development.py` file to migrate, run the following command:
+With the database created and python requirements installed we can migrate the tables. We will use the `orator_config.py` file to migrate, run the following command:
 
 ```bash
-$ orator migrate -c orator_development.py
+$ orator migrate -c orator_config.py
 ```
 
-If you want to recreate a fresh database. Just delete all the tables from the database and repeat the steps above. 
-
-### .env file
-
-Using the `.template.env` fill the missing keys and save it as a **new** copy with the name `.env` at the root directory of the project. Pipenv will use this file to load the environment variables
+If you want to begin a new database, delete all the tables and run the migrations again.
```

Comment:
```suggestion
If you want to begin anew, just recreate the database and run the migrations again. You can do that by running `dropdb neo-data` and creating the database again as instructed above.
```

Reply:
Not sure if you need to drop the database. I think that just deleting the tables is enough. Do you have a reason for this?

Reply:
We already discussed this. If you say you want to begin a new database, it means you're destroying one and creating another. If not, then you're only deleting the tables from one database, which is much more cumbersome (as you have to go through them one by one) than merely deleting the datatabase.

Reply:
I see how it could be more trouble to delete one by one but maybe if we run something like this:
```
DROP SCHEMA public CASCADE;
CREATE SCHEMA public
```

then it would delete every table without deleting the database. Do you think this makes sense?